.github/workflows/deploy.yml

```
@@ -26,27 +26,24 @@ jobs:
           jupyter lite build
       - name: Add custom index.html (live.sympy.org landing page) and static/ files
         run: |
-          pip install jinja2 babel
-          ls -ltr ./_output
           ./generateindex.py
-          ls -ltr ./_output
       - name: Add CNAME file containing live.sympy.org
         run: |
           echo "live.sympy.org" > ./_output/CNAME
       - name: Upload (dist)
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: jupyterlite-demo-dist-${{ github.run_number }}
           path: ./_output
 
   deploy:
     if: github.ref == 'refs/heads/main'
-    needs: [build]
+    needs: build
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2.3.1
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@v4
         with:
           name: jupyterlite-demo-dist-${{ github.run_number }}
           path: ./dist
```

```suggestion
      - uses: actions/download-artifact@v4
        with:
          name: jupyterlite-demo-dist-${{ github.run_number }}
          path: ./dist
          merge-multiple: true
```

Bumping to `download-artifact@v4` means that the directory structure will be different, so without the above option, the artifacts will be downloaded to a folder `dist/jupyterlite-demo-dist-${{ github.run_number }}`, instead of them being in `dist/` directly.

---

Oh wow, thanks for catching that!
You probably saved me an hour of debugging and trying to figure this out ;)